packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -1404,20 +1410,22 @@ class OverlayPortalController {
 /// [OverlayPortalController.show] was called. The last [OverlayPortal] to have
 /// called `show` gets to paint its overlay child in the foreground.
 ///
+/// {@template flutter.widgets.overlayPortalVsOverlayEntry}
 /// ### Differences between [OverlayPortal] and [OverlayEntry]
 ///
 /// The main difference between [OverlayEntry] and [OverlayPortal] is that
 /// [OverlayEntry] builds its widget subtree as a child of the target [Overlay],
-/// while [OverlayPortal] uses [overlayChildBuilder] to build a child widget of
-/// itself. This allows [OverlayPortal]'s overlay child to depend on the same
-/// set of [InheritedWidget]s as [OverlayPortal], and it's also guaranteed that
-/// the overlay child will not outlive its [OverlayPortal].
+/// while [OverlayPortal] uses [OverlayPortal.overlayChildBuilder] to build a
```

Comment:
Does dart doc not recognize references to a local instance method if the class name is omitted?

Reply:
Good call, I do not think it is necessary

```suggestion
/// while [OverlayPortal] uses [overlayChildBuilder] to build a
```

Reply:
yeah my bad. idk why i thought that was needed.

Reply:
I think I know why now. 🙃 
Since it is in a template, where it is reused is outside of OverlayPortal. That's why the tests are failing now.

Reply:
Ohh right! That's why I had added that. xD